src/scss/components/help-bar/_help-bar.scss

```diff
@@ -166,3 +166,8 @@
     overflow-x: clip;
   }
 }
+
+// help bar to very top
+.add-more-space-to-top {
```

Comment:
You might want to add another specificity so that it is guaranteed  to apply. Even something like `body .add-more-space-to-top`

Reply:
I updated the webpace, yarn, and info.yml file. Let me know what you think.